src/components/strategies/create/BuySellBlock/InputLimit.tsx

```diff
@@ -1,10 +1,12 @@
-import { useFiatCurrency } from 'hooks/useFiatCurrency';
-import { Token } from 'libs/tokens';
 import { ChangeEvent, FC, FocusEvent } from 'react';
 import { carbonEvents } from 'services/events';
+import { useFiatCurrency } from 'hooks/useFiatCurrency';
```

Comment:
why keep reordering imports?

Reply:
will stop doing it manually. Maybe will add in the future eslint rule for it